src/printer.js

```diff
@@ -3846,6 +3846,14 @@ function nodeStr(node, options) {
     ? alternate.quote
     : preferred.quote;
 
+  // Directives are exact code unit sequences, which means that you can't
+  // change the escape sequences they use.
+  // See https://github.com/prettier/prettier/issues/1555
+  // and https://tc39.github.io/ecma262/#directive-prologue
+  if (node.type === 'DirectiveLiteral') {
+    return raw;
+  }
```

Comment:
Good point, I just pushed up a fix at https://github.com/prettier/prettier/pull/1560/commits/5d338eb71baea71f0faddb14dd959bd0effa2458

It looks like that also fixes the tests, @azz (`npm test -- -u` didn't seem to do anything for me...)